cmd/qos.go

```diff
@@ -58,11 +64,14 @@ func getServiceQoSInstances(
 			if _, ok := hydratorServiceIDsIdx[serviceID]; ok {
 				hydratorQoSGenerators[serviceID] = evmEndpointStore
 			}
-		case config.ServiceIDSolana:
+
+		case serviceIDSolana:
```

Comment:
The logic here is complex enough to justify using a builder/factory function pattern here. At-least having something like `func buildSolanaQoSInstance(...)` in a `solana.go` file either here or under `config` package will make the initialization/configuration code easier to read and maintain.

Reply:
This feels out of scope for this PR as this PR only intends to add the functionality to support multiple service IDs per `QoSService`.

I've added a `TODO` to capture your comment here.